### DISCUSSION TITLE
N64 8bitdo modkit

### DISCUSSION THREAD

**Discussion by SurrealCerebralCereal** [darthcloud/BlueRetro#934] [General]
I was reading this is now compatible in D mode but it does not work on the latest firmware. in D mode the up stick acts like its pressed all the time no matter what. I did mess with some of the tolerances in the advanced settings and it worked once after restarting, but it goes back to the up press on the stick. (this is with the hall effect joystick that comes with the modkit, the original stick doesn't have issues weirdly. The same hall effect joystick works fine in the original n64 controller board. It also works fine on the switch and an intec bluetooth adapter I have. Its just blue retro D mode that has issues.) S mode works on first pairing actually just fine but if you power off the console and back on and reconnect, the A and B buttons are switched.  Just wondering what I can do. I really like these adapters.

**Comment by darthcloud** [comment]
With 8bitdo controller you need to press a digital button a couple time after connection to BlueRetro while leaving the stricks and trigger at neutral position. This allow BlueRetro to calibrate the center.

Dont touch the joystick, connect, and press A a few times.

I tested mine with v1.9.2 and I had no issue
